homeassistant/components/bosch_shc/cover.py

```diff
@@ -0,0 +1,95 @@
+"""Platform for cover integration."""
+from boschshcpy import SHCSession, SHCShutterControl
+
+from homeassistant.components.cover import (
+    ATTR_POSITION,
+    DEVICE_CLASS_SHUTTER,
+    SUPPORT_CLOSE,
+    SUPPORT_OPEN,
+    SUPPORT_SET_POSITION,
+    SUPPORT_STOP,
+    CoverEntity,
+)
+
+from .const import DATA_SESSION, DOMAIN
+from .entity import SHCEntity
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the SHC cover platform."""
+
+    entities = []
+    session: SHCSession = hass.data[DOMAIN][config_entry.entry_id][DATA_SESSION]
+
+    for cover in session.device_helper.shutter_controls:
+        entities.append(
+            ShutterControlCover(
+                device=cover,
+                parent_id=session.information.unique_id,
+                entry_id=config_entry.entry_id,
+            )
+        )
+
+    if entities:
+        async_add_entities(entities)
+
+
+class ShutterControlCover(SHCEntity, CoverEntity):
+    """Representation of a SHC shutter control device."""
+
+    @property
+    def supported_features(self):
+        """Flag supported features."""
+        return SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_STOP | SUPPORT_SET_POSITION
+
+    @property
+    def device_class(self):
+        """Return the class of this device, from component DEVICE_CLASSES."""
+        return DEVICE_CLASS_SHUTTER
```

Comment:
Use entity class attribute instead entity property methods that are returning a constant. 

```suggestion
class ShutterControlCover(SHCEntity, CoverEntity):
    """Representation of a SHC shutter control device."""
    
    _attr_device_class = DEVICE_CLASS_SHUTTER
    _attr_supported_features = (
        SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_STOP | SUPPORT_SET_POSITION
    )
```